script/load_templates

```diff
@@ -91,18 +92,20 @@ GetOptions(\%options, "apibase=s", "apikey=s", "apisecret=s", "clean", "host=s",
 
 usage(1) if $options{help} || $#ARGV;
 
-my $datafile = shift @ARGV;
-die "Data file not found\n" unless $datafile && -r $datafile;
+# Slurp file specified as an argument or STDIN in case of "-"
+my $datafile = join("", <>);
+die "Data file not found\n" unless length $datafile;
 
 my $info;
 
-if ($datafile =~ /\.json$/) {
-    local $/;
-    open(my $fh, '<', $datafile);
-    $info = Cpanel::JSON::XS->new->relaxed->decode(<$fh>);
-    close $fh;
+if ($datafile =~ m/{/) {
+    $info = Cpanel::JSON::XS->new->relaxed->decode($datafile);
     dd $info;
 }
+elsif ($datafile =~ m/products:/) {
+    $info = $datafile;
+    print $info . "\n";
```

Comment:
We use a Modern Perl dialect. You can do `use Mojo::Base -strict;` in the beginning of the script and `say $info;` here instead of the `print`. 😉